snapcraft/cli/store.py

```diff
@@ -497,13 +497,16 @@ def close(snap_name, channels):
 @click.option(
     "--arch", metavar="<arch>", help="The snap architecture to get the status for"
 )
+@click.option("--track", metavar="<track>", help="The snap track to get the status for")
```

Comment:
what if we make it a list of tracks? Would that be useful?

Reply:
Definitely, I just wanted to match the architecture there. Would you like to see just `multiple=True`? Should we introduce the same for `--arch`? Should we also introduce short options, or would you like to see it spelled out each time?

Reply:
Let's go with `multiple=True`, this would give us a no syntax change feature in the future for `--arch`.
Let's leave short options out for now, those require some bike shedding I am not prepared to do so close to the end of the year :-)

Thanks for working on this!

Reply:
Alright will do.

Reply:
Done.